pkg/injection/otel/otel.go

```diff
@@ -0,0 +1,15 @@
+package otel
+
+import (
+	webhookotel "github.com/Dynatrace/dynatrace-operator/pkg/webhook/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func SpanOptions(opts ...trace.SpanStartOption) []trace.SpanStartOption {
+	options := make([]trace.SpanStartOption, 0)
+	options = append(options, opts...)
+	options = append(options, trace.WithAttributes(
+		attribute.String(webhookotel.WebhookPodNameKey, webhookotel.GetWebhookPodName())))
+	return options
+}
```

Comment:
is this the correct place for this?

the `injection` directory is for packages that are related to the injection of the user workloads, this is more of just `instrumentation` of otel for our webhook

or am I missing something?

Reply:
The idea is to have package specific otel related stuff close to the resp. package but nevertheless separated. Previously I've implemented package specific `StartSpan` functions, but that didn't fly with the automatic span naming.

Reply:
we could put it into `pkg/injection/internal/otel`, so it limits the public stuff

Reply:
[8d47858](https://github.com/Dynatrace/dynatrace-operator/pull/2289/commits/8d4785872b1570310f4f1f3e084a692d52e52202)